runtimes/typescript/lbr-prelude/src/LambdaBuffers/Runtime/Prelude.ts

```diff
@@ -0,0 +1,700 @@
+import type { Json } from "./Json.js";
+import { JsonError, Scientific } from "./Json.js";
+import * as LbJson from "./Json.js";
+import type { Eq } from "./Eq.js";
+import type { Ord } from "./Ord.js";
+import { Buffer } from "node:buffer";
```

Comment:
I don't think this will work in the browser. Keep in mind we need to keep this available to web AND node envs.

Reply:
Please check and comment.